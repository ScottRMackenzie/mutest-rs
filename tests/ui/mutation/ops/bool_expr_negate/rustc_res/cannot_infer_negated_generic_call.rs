//@ fail
//@ stderr

fn f() {
    let _: bool = !Default::default();
}

#[test]
fn test() {
    f();
}
