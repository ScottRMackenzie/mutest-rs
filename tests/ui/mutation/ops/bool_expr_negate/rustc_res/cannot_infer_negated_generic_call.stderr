error[E0282]: type annotations needed
 --> tests/ui/mutation/ops/bool_expr_negate/rustc_res/cannot_infer_negated_generic_call.rs:5:19
  |
5 |     let _: bool = !Default::default();
  |                   ^^^^^^^^^^^^^^^^^^^ cannot infer type

error: aborting due to 1 previous error

For more information about this error, try `rustc --explain E0282`.
